src/sbt-test/travisci/build-matrix/.travis.yml

```diff
@@ -7,3 +7,5 @@ matrix:
   include:
    - scala: 2.12.0
      jdk: oraclejdk8
+   - scala: 2.12.0
+     jdk: oraclejdk9
```

Comment:
Wouldn't it be better to jump straight to 10? `oraclejdk10` is now supported by Travis.